src/particles/elements/Drift.H

```diff
@@ -123,6 +130,9 @@ namespace impactx
             py = pyout;
             pt = ptout;
 
+            // apply transverse aperture
+            apply_aperture(x, y, idcpu);  //value of idcpu?
```

Comment:
`idcpu` is passed above into the function, alongside the other particle attributes. All arguments passed to this function are set before the element is called.
```suggestion
            apply_aperture(x, y, idcpu);
```